server/server/src/securityStorage.ts

```diff
@@ -0,0 +1,89 @@
+//
+// Copyright © 2021 Anticrm Platform Contributors.
+//
+// Licensed under the Eclipse Public License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License. You may
+// obtain a copy of the License at https://www.eclipse.org/legal/epl-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import core, { Class, Doc, Space, DocumentQuery, Ref, Storage, Tx, Account, TxCreateDoc, TxAddCollection, Emb, TxProcessor, makeEmb, Member, Hierarchy } from '@anticrm/core'
+
+export class SecurityStorage {
```

Comment:
I think we should separate into two stores, one fill fill and update model handling `tx` and second one will filter user requests. Please refer to #101 for https://github.com/hardcoreeng/platform/pull/101/files#diff-a98d95ae7d7f872cd55f123e72db9403e4ec76f5c11ea12b8a8d2f25a0e705be